extension/content.js

```diff
@@ -7,8 +7,9 @@ const isDashboard = () => location.pathname === '/' || /(^\/(dashboard))/.test(l
 const isRepo = () => /^\/[^/]+\/[^/]+/.test(location.pathname);
 const isRepoRoot = () => location.pathname.replace(/\/$/, '') === `/${ownerName}/${repoName}` || /(\/tree\/)(\w|\d|\.)+(\/$|$)/.test(location.href);
 const isPR = () => /^\/[^/]+\/[^/]+\/pull\/\d+/.test(location.pathname) || /^\/[^/]+\/[^/]+\/pull\/\d+\/commits\/[0-9a-f]{5,40}/.test(location.pathname);
+const isPRCommit = () => /^\/[^/]+\/[^/]+\/pull\/\d+\/commits\/[0-9a-f]{5,40}/.test(location.pathname);
 const isCommit = () => {
```

Comment:
I'm OK with this as long as we don't use the name `utils`, but something more explicit and with one purpose only like `urlHelper`, `urlUtils` or just `url`.